qa/workunits/rados/test_envlibrados_for_rocksdb.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
looks good, the exit status should indicate if it fails - I like using '#!/bin/bash -ex' for this so we have logging of what precisely failed if something goes wrong